prov/efa/src/efa.h

```diff
@@ -164,7 +165,7 @@ struct efa_domain {
 	struct util_domain	util_domain;
 	enum efa_domain_type	type;
 	struct fid_domain	*shm_domain;
-	struct efa_context	*ctx;
+	struct efa_device	*device;
```

Comment:
I find `efa_device` actually replaces both `efa_context` and `efa_device_attr`. Probably mentioning it as well in the commit message. The current one makes me think `efa_device` only replaces `efa_context`.

Reply:
Thank you! Updated commit message in new revision.